### DISCUSSION TITLE
Disable vendor prefixes

### DISCUSSION THREAD

**Discussion by mustafa0x** [windicss/windicss#118] [Q&A] [answered]
I've started using this (using the svelte plugin) and I see in bundle.css: `.windi-18feqd2{display:-webkit-box;display:-ms-flexbox;display:-webkit-flex;display:flex}`.

How can I stop windicss from adding vendor prefixes? My project doesn't require supporting old browsers, so this increases the CSS size unnecessarily.

**Comment by voorjaar** [comment, marked as answer]
Reference https://github.com/windicss/windicss/issues/132

**Reply by voorjaar** [reply]
add this in your config file

```js
// tailwind.config.js
module.exports = {
  prefixer: false,
  theme: {
     ...
  }
}
```

**Reply by mustafa0x** [reply]
Thanks!
